stackhpc_ipa_hardware_managers/system_nic.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2015 Rackspace, Inc.
+# Copyright 2018 StackHPC Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import glob
+import os
+
+from ironic_python_agent import errors
+from ironic_python_agent import hardware
+from ironic_python_agent import utils
+
+from oslo_concurrency import processutils
+from oslo_log import log
+
+UEVENT_PCI_SLOT_NAME_PREFIX = "PCI_SLOT_NAME="
```

Comment:
nit: could prefix this with _ to indicate it's private

Reply:
fair point